### DISCUSSION TITLE
Changing currency for products whose price is set by a wrong currency

### DISCUSSION THREAD

**Discussion by Megidd** [saleor/saleor#7848] [Q&A] [answered]
# Change currency

I can change the product currency by `export DEFAULT_CURRENCY="USD"` command. The problem is the environment variable only works for the new products which are going to be created. The old products would be displayed on dashboard with the old currency before the change.

![Screenshot_20210816_202804](https://user-images.githubusercontent.com/17475482/129594593-50acf1e5-117e-4db5-bb3f-b22319147c79.png)

# How

How can I change the currency for old products. I even edited the PostgreSQL database table currency column for products, but it didn't work! Somewhere it should be possible to change the currency for old products. Right? Thanks :smile: 

I'm using version `2.11`.

**Comment by karolkielecki** [comment, marked as answer]
@Megidd Hi! As far as I remember, it wasn't possible to change the currency for previously created products in 2.11. We're currently working on **Saleor 3.0**, which introduces multichannel w/ multicurrency. We recommend you using the latest beta version--it will be easier for you to manage products, etc. If you update your project to 3.0, you will be able to create multiple channels (each with a specific currency) and assign products to any channels you want.
